### PR TITLE
addscore endpoint

### PR DESCRIPTION
**Pull request by yuna-iwata**
adding score endpoint created

### CODE REVIEW

File: api.py [hunk 1]
```diff
@@ -12,6 +12,22 @@
 def index():
     return 'Dino'
 
+
+@app.route('/submit-score', methods=['POST'])
+def submit_score():
+    data = request.json
+    username = data["username"]
+    score = data["score"]
+    print('score',score)
```
Comment (louisglan) [inline api.py]: remove this print

File: api.py [hunk 2]
```diff
@@ -12,6 +12,22 @@
...
+    userid = select("SELECT user_id FROM users WHERE username = %s", (username,))
+    formatuserid = userid[0][0]
+    print(userid[0][0])
```
Comment (louisglan) [inline api.py]: remove this print

File: api.py [hunk 3]
```diff
@@ -12,6 +12,22 @@
...
     return 'Dino'
 
+
+@app.route('/submit-score', methods=['POST'])
+def submit_score():
+    data = request.json
+    username = data["username"]
+    score = data["score"]
+    print('score',score)
+    userid = select("SELECT user_id FROM users WHERE username = %s", (username,))
```
Comment (louisglan) [inline api.py]: userid, formatuserid could be user_id, format_user_id

File: api.py (hunk 1 above)
Comment (yuna-iwata) [reply api.py]: okeee

**Review by louisglan — CHANGES_REQUESTED** [review]
Requested a couple print removal and formatting changes

File: api.py (hunk 3 above)
Comment (yuna-iwata) [reply api.py]: nice

File: api.py [hunk 4]
```diff
@@ -12,6 +12,20 @@
...
     return 'Dino'
 
+
+@app.route('/submit-score', methods=['POST'])
+def submit_score():
+    data = request.json
+    username = data["username"]
+    score = data["score"]
+    userid = select("SELECT user_id FROM users WHERE username = %s", (username,))
+    format_user_id = userid[0][0]
```
Comment (louisglan) [inline api.py:22]: userid could be user_id (and on above line)

File: api.py [hunk 5]
```diff
@@ -12,6 +12,20 @@
...
+    query = insert(
+        """INSERT INTO scores (user_id, score, date) 
+        VALUES (%s, %s, current_timestamp);""", 
+        (formatuserid,score))
```
Comment (louisglan) [inline api.py]: formatuserid should be format_user_id I think

File: api.py (hunk 4 above)
Comment (yuna-iwata) [reply api.py:22]: 👍

**Review by louisglan — APPROVED** [review]
Fabulous!

**Pull request merged**